### PR TITLE
[HID-2387] user counts

### PR DESCRIPTION
**Pull request by rupl**
# HID-2387

Admin-only endpoint for aggregate user counts.

### Testing

- `GET /api/v3/numbers`
- No token or non-admin token should 403
- Bad token should 401
- Admin token should 200 with a JSON object:

```json
{
  "totalUsers": 321,
  "emailVerified": 123,
}
```

### CODE REVIEW

File: api/controllers/NumbersController.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/**
+ * @module NumbersController
+ * @description Provides aggregate counts for: users.
+ */
+const Boom = require('@hapi/boom');
```
Comment (cafuego) [inline api/controllers/NumbersController.js:5]: 💥

**Review by cafuego — APPROVED** [review]
Assuming I can make auth for for n8n, this ought to do the trick for reporting.

**Review by orakili — APPROVED** [review]

**Comment by orakili** [comment]
Note:  approved based on the code review but I'm still having unrelated issues that prevent me from testing locally.

**Comment by rupl** [comment]
That's ok, we can put it on dev and test it. I tried to set up the unit tests for the basics: missing, invalid, or non-admin tokens. We saw how that PR is going, but it might be fixable without adding a DB export to CI.

@cafuego I think for n8n the easiest thing to do it to make a bot account. More details in ticket.

**Pull request merged**